File: Iniciante/URI 1011.kt
import java.util.Scanner;

val pi: Double = 3.14159;

fun main(args: Array<String>) {
    val leitor = Scanner(System.`in`);
    
    val raio: Double = leitor.nextDouble();
    val volume: Double = (4.0 / 3) * pi * Math.pow(raio, 3.0);
    
    println(String.format("VOLUME = %.3f", volume));
}

File: Iniciante/URI 1011.scala
import java.util.Scanner

object Main {
    
    private final val pi = 3.14159

    def main(args: Array[String]) {
        val leitor = new Scanner(System.in)
        val raio = leitor.nextDouble()
        val volume = (4.0 / 3) * pi * Math.pow(raio, 3.0)
        printf("VOLUME = %.3f\n", volume)
    }
	
}